src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+
+const dataAtributToNumber = (salary) => {
+  return salary.replace(/\D/g, '');
+};
+
+const sortList = (list) => {
+  list
+    .toSorted(
+      (value1, value2) =>
+        dataAtributToNumber(value2.dataset.salary) -
+        dataAtributToNumber(value1.dataset.salary),
+    )
+    .forEach((employee, i) => list[i].closest('ul').appendChild(employee));
```

Comment:
The 'closest' method is not necessary here. Since each 'li' is a direct child of 'ul', you can use 'list[i].parentElement.appendChild(employee)' instead.